static/app/views/performance/newTraceDetails/traceWarnings/performanceSetupWarning.tsx

```diff
@@ -0,0 +1,237 @@
+import {useEffect, useMemo} from 'react';
```

Comment:
Reused most of the code from old banner: https://github.com/getsentry/sentry/blob/master/static/app/views/performance/traceDetails/content.tsx#L428